src/Components/ApiFetch/RoversApiFetch.tsx

```diff
@@ -0,0 +1,15 @@
+﻿import React from "react";
+
+export interface RoverImages {
```

Comment:
Am I right in thinking this is just a single image? If so, it's a bit confusing for the interface name to be plural.

Reply:
Good point! Changed to singular.